### ISSUE TITLE
[Linux] Dota 2 not starting up correctly on Ubuntu 15.10

### ISSUE THREAD

**Issue by icyphox** [ValveSoftware/Dota-2#832]
I've installed Dota 2 on my Ubuntu box, and the Nvidia proprietary drivers too. And now, when I launch Dota, I get a HUGE logo of Dota on the screen and then two black boxes appear over that. And that's it, it's hung. Any ideas? Here's what it looks like: http://imgur.com/Jv8aj1B
- Okay so here's my kernel version: `4.2.0-16-generic`
- And, here's my driver version: `304.131`
- Finally, I have a GT 630 2GB DDR3.

**Comment by gdrewb-valve** [comment]
The first thing to try is to update your drivers.

**Comment by icyphox** [comment]
I think I have the latest drivers because I used `sudo apt-get install nvidia-current` to install them.

**Comment by gdrewb-valve** [comment]
The current NVIDIA drivers are at 361.42, the 304 series was from quite a while ago.

**Comment by icyphox** [comment]
Okay, I'll try installing those and get back to you.
EDIT: I actually had installed these before, now that I check my Downloads folder. I remember now, it caused some severe display issues, hence I had to purge them. Maybe compatibility issues.

EDIT2: Okay! I installed 340.32 and it worked! I know it's not the latest, but I'm not going to bother updating it. Because, if it ain't broke, don't fix it. :D Cheers.
